server/api/student/index.js

```diff
@@ -2,11 +2,13 @@ import {AsyncRouter} from 'express-async-router';
 import * as controller from './student.controller';
 import {hasRole} from '../../auth/auth.service';
 import paginate from 'express-paginate';
-
+import multiparty from 'connect-multiparty';
+var multipartyMiddleware = multiparty();
```

Comment:
Use const for variables that are not gonna change
In ES6 in general we don't use `var` but  `const` and `let`